ang/crmMosaico/TemplateItem.js

```diff
@@ -2,7 +2,8 @@
   // "crmMosaicoTemplateItem" displays an actionable thumbnail.
   // To specify action handlers, use `on-item-click`, `on-item-preview`, etal.
   // To conditionally enable/disable actions, use `check-item-click`, `check-item-preview`, etal.
-  // Example usage: <div crm-mosaico-template-item="{title: 1, subtitle: 2, thumbnail: 3}" on-item-click="alert('Click')" on-item-preview="alert('Preview')"></div>
+  // Example usage: <div crm-mosaico-template-item="{state: 'selected', title: 1, subtitle: 2, thumbnail: 3}" on-item-click="alert('Click')" on-item-preview="alert('Preview')"></div>
+  // Avaiable state options are (select, selected, configure, new)
```

Comment:
this has been fixed in 

https://github.com/compucorp/uk.co.vedaconsulting.mosaico/pull/5/commits/99b27c334a1d8566afe9852593ff77f94080c3fe